src/Mod/Draft/draftguitools/gui_scale.py

```diff
@@ -108,19 +108,24 @@ def proceed(self):
         _msg(translate("draft", "Pick base point"))
 
     def set_ghosts(self):
-        """Set the previews of the objects to scale."""
-        if self.ui.isSubelementMode.isChecked():
-            return self.set_subelement_ghosts()
-        self.ghosts = [trackers.ghostTracker(self.selected_objects)]
+        """Set the ghost to display."""
+        for ghost in self.ghosts:
+            ghost.remove()
+        if self.task and self.task.isSubelementMode.isChecked():
+            self.ghosts = self.get_subelement_ghosts()
+        else:
+            self.ghosts = [trackers.ghostTracker(self.selected_objects)]
 
-    def set_subelement_ghosts(self):
-        """Set the previews of the subelements from an object to scale."""
+    def get_subelement_ghosts(self):
+        """Get ghost for the subelements (vertices, edges)."""
         import Part
```

Comment:
Move to top?

Reply:
I don't know. That code was already there.